src/styles/globals.css

```diff
@@ -116,8 +9,8 @@ a {
   left: 0;
   border-radius: 3px;
   display: none;
```

Comment:
background color for search option box can be changed to whatever color matches the color scheme you chose